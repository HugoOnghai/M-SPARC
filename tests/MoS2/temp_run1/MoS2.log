
                            < M A T L A B (R) >
                  Copyright 1984-2019 The MathWorks, Inc.
                  R2019a (9.6.0.1072779) 64-bit (glnxa64)
                               March 8, 2019

 
To get started, type doc.
For product information, visit www.mathworks.com.
 

 Reading .inpt file ...


<INPT>
# $ cat MoS2.inpt
# nprocs: 48
# Test: MoS2 #
LATVEC:
1 0 0
-0.5 0.8666 0
0 0 1
CELL: 6.1 6.1 30
MESH_SPACING: 0.20
FD_ORDER: 12
BC: P P D
KPOINT_GRID: 3 3 1
EXCHANGE_CORRELATION: GGA_PBE
ELEC_TEMP_TYPE: fermi-dirac
ELEC_TEMP: 315.773
TOL_SCF: 1e-6
#ELEC_TEMP: 
CALC_STRESS: 1
PRINT_FORCES: 1
PRINT_ATOMS: 0
MIXING_VARIABLE: potential
MIXING_PRECOND: none

# MD
#MD_FLAG:  1
#MD_METHOD: NVT_NH
#MD_TIMESTEP: 1
#MD_NSTEP: 10
#PRINT_RESTART_FQ: 3
#RESTART_FLAG: 0

<\INPT>

 Reading .ion file ...
 Number of atom types : 2
 Total number of atoms: 3
 Default atomic mass for Mo is 95.950000
 Default atomic mass for S is 32.067500
 pseudo_dir # 1 = Mo.psp8
 pseudo_dir # 2 = S.psp8

 COORD:
   0.070149695000000   3.296864329500000  15.000000000000000
   2.806003050000000   1.762084904600000  18.655220000000000
   3.050003050000000   1.762084904600000  12.344780000000000

   5.403031739800300

   0.031202687423158

 kpoint grid before symmetry:
  -0.343343459408721  -0.343343459408721                   0
                   0  -0.343343459408721                   0
   0.343343459408721  -0.343343459408721                   0
  -0.343343459408721                   0                   0
                   0                   0                   0
   0.343343459408721                   0                   0
  -0.343343459408721   0.343343459408721                   0
                   0   0.343343459408721                   0
   0.343343459408721   0.343343459408721                   0

 kpoint grid after symmetry:
                   0                   0                   0
   0.343343459408721                   0                   0
  -0.343343459408721   0.343343459408721                   0
                   0   0.343343459408721                   0
   0.343343459408721   0.343343459408721                   0

## Chebyshev polynomial degree not provided, finding npl ...
## Based on the mesh size, npl is set to: 35
## Number of states not provided, finding Nev ...
## Based on the number of electrons, Nev is set to: 21
## Poisson tolerance not provided, choosing poisson_tol ...
## poisson_tol is set to: 1.000e-08
## Pseudocharge tolerance not provided, choosing pseudocharge_tol ...
## pseudocharge_tol is set to: 1.000e-08
 Creating differentiation matrices ...
 Done. (5.601 sec)

 Estimated memory usage:
 Total:    1.22 GB
 orbitals            :  870.94 MB
 sparse matrices     :  350.45 MB
 global-size vectors :   14.52 MB
 mixing histories    :   14.52 MB

 Finding rb for Mo ...
 rb = {2.843 2.800 2.800}, int_b = -14.001092001547342, err_rb = 1.092e-03
 rb = {3.043 3.000 3.000}, int_b = -14.000106116115139, err_rb = 1.061e-04
 rb = {3.243 3.200 3.200}, int_b = -14.000131249724772, err_rb = 1.312e-04
 rb = {3.443 3.400 3.400}, int_b = -14.000095516067914, err_rb = 9.552e-05
 rb = {3.643 3.600 3.600}, int_b = -14.000083817564356, err_rb = 8.382e-05
 rb = {3.843 3.800 3.800}, int_b = -14.000069869448565, err_rb = 6.987e-05
 rb = {4.043 4.000 4.000}, int_b = -14.000059491311532, err_rb = 5.949e-05
 rb = {4.243 4.200 4.200}, int_b = -14.000052154535517, err_rb = 5.215e-05
 rb = {4.443 4.400 4.400}, int_b = -14.000047442631258, err_rb = 4.744e-05
 rb = {4.643 4.600 4.600}, int_b = -14.000043593243456, err_rb = 4.359e-05
 rb = {4.843 4.800 4.800}, int_b = -14.000041396370371, err_rb = 4.140e-05
 rb = {5.043 5.000 5.000}, int_b = -14.000036833104586, err_rb = 3.683e-05
 rb = {5.243 5.200 5.200}, int_b = -14.000032829286834, err_rb = 3.283e-05
 rb = {5.443 5.400 5.400}, int_b = -14.000029074702088, err_rb = 2.907e-05
 rb = {5.643 5.600 5.600}, int_b = -14.000028177819640, err_rb = 2.818e-05
 rb = {5.843 5.800 5.800}, int_b = -14.000024139750005, err_rb = 2.414e-05
 rb = {6.043 6.000 6.000}, int_b = -14.000018233509889, err_rb = 1.823e-05
 rb = {6.243 6.200 6.200}, int_b = -14.000016169134048, err_rb = 1.617e-05
 rb = {6.443 6.400 6.400}, int_b = -14.000014665743539, err_rb = 1.467e-05
 rb = {6.643 6.600 6.600}, int_b = -14.000013146278974, err_rb = 1.315e-05
 rb = {6.843 6.800 6.800}, int_b = -14.000014001197700, err_rb = 1.400e-05
 rb = {7.043 7.000 7.000}, int_b = -13.999998243908331, err_rb = 1.756e-06
 rb = {7.243 7.200 7.200}, int_b = -14.000000370349266, err_rb = 3.703e-07
 rb = {7.443 7.400 7.400}, int_b = -13.999999927864048, err_rb = 7.214e-08
 rb = {7.643 7.600 7.600}, int_b = -14.000000009977800, err_rb = 9.978e-09
 rb = {7.643 7.600 7.600}
 Finding rb for S ...
 rb = {2.437 2.400 2.400}, int_b = -5.999383434375397, err_rb = 6.166e-04
 rb = {2.637 2.600 2.600}, int_b = -5.999772222126428, err_rb = 2.278e-04
 rb = {2.837 2.800 2.800}, int_b = -5.999946952438726, err_rb = 5.305e-05
 rb = {3.037 3.000 3.000}, int_b = -6.000008812294018, err_rb = 8.812e-06
 rb = {3.237 3.200 3.200}, int_b = -6.000028706826659, err_rb = 2.871e-05
 rb = {3.437 3.400 3.400}, int_b = -6.000031642035080, err_rb = 3.164e-05
 rb = {3.637 3.600 3.600}, int_b = -6.000029569054694, err_rb = 2.957e-05
 rb = {3.837 3.800 3.800}, int_b = -6.000025373947377, err_rb = 2.537e-05
 rb = {4.037 4.000 4.000}, int_b = -6.000021072256589, err_rb = 2.107e-05
 rb = {4.237 4.200 4.200}, int_b = -6.000018028386865, err_rb = 1.803e-05
 rb = {4.437 4.400 4.400}, int_b = -6.000016480757174, err_rb = 1.648e-05
 rb = {4.637 4.600 4.600}, int_b = -6.000015507214554, err_rb = 1.551e-05
 rb = {4.837 4.800 4.800}, int_b = -6.000015677565834, err_rb = 1.568e-05
 rb = {5.037 5.000 5.000}, int_b = -6.000014106073255, err_rb = 1.411e-05
 rb = {5.237 5.200 5.200}, int_b = -6.000013008447124, err_rb = 1.301e-05
 rb = {5.437 5.400 5.400}, int_b = -6.000012139656846, err_rb = 1.214e-05
 rb = {5.637 5.600 5.600}, int_b = -6.000013363002438, err_rb = 1.336e-05
 rb = {5.837 5.800 5.800}, int_b = -6.000012190334539, err_rb = 1.219e-05
 rb = {6.037 6.000 6.000}, int_b = -6.000009339949510, err_rb = 9.340e-06
 rb = {6.237 6.200 6.200}, int_b = -6.000009090628247, err_rb = 9.091e-06
 rb = {6.437 6.400 6.400}, int_b = -6.000009063825310, err_rb = 9.064e-06
 rb = {6.637 6.600 6.600}, int_b = -6.000008802895501, err_rb = 8.803e-06
 rb = {6.837 6.800 6.800}, int_b = -6.000010104718418, err_rb = 1.010e-05
 rb = {7.037 7.000 7.000}, int_b = -5.999998728455862, err_rb = 1.272e-06
 rb = {7.237 7.200 7.200}, int_b = -6.000000268230179, err_rb = 2.682e-07
 rb = {7.437 7.400 7.400}, int_b = -5.999999947891073, err_rb = 5.211e-08
 rb = {7.637 7.600 7.600}, int_b = -6.000000007172971, err_rb = 7.173e-09
 rb = {7.637 7.600 7.600}

 ###############################################################
 Relaxation step number: 1 

 Starting pseudocharge generation and self energy calculation...
 Integration b = 26.000000007599

 Integration b_ref = 26.000000000014

 ****************************************
 *          Eself_ref = 478.738373       *
 ****************************************
 Done. (4.304911 s)
 Time for b calculation: 4.320 seconds.

 Starting calculating nonlocal projectors ...
 Done. (2.418775 s)

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Starting SCF iteration...
 AAR converged to a relative residual of 8.61184e-09 in 277 iterations.
 Poisson problem took 29.071520s
 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 1 
 ============================================= 
 Fermi energy = 4.561984
 This SCF iteration took 214.312 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 2 
 ============================================= 
 Fermi energy = 0.331362
 This SCF iteration took 180.428 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 3 
 ============================================= 
 Fermi energy = -0.147006
 This SCF iteration took 177.618 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 4 
 ============================================= 
 Fermi energy = -0.199066
 AAR converged to a relative residual of 7.92921e-09 in 229 iterations.
 Poisson problem took 23.914305s
 Escc = 1.01583044
 Etot = -89.00848756
 Eatom = -29.66949585
 Error in SCF iteration: 1.2932e-01 
 This SCF iteration took 201.700 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  2 
 ========================= 
 Fermi energy = -0.193412
 AAR converged to a relative residual of 9.31722e-09 in 295 iterations.
 Poisson problem took 30.719763s
 Escc = -0.79474076
 Etot = -88.98052940
 Eatom = -29.66017647
 Error in SCF iteration: 3.6204e-01 
 This SCF iteration took 207.953 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  3 
 ========================= 
 Fermi energy = -0.200492
 AAR converged to a relative residual of 9.76228e-09 in 301 iterations.
 Poisson problem took 31.693883s
 Escc = 1.48964671
 Etot = -89.08072041
 Eatom = -29.69357347
 Error in SCF iteration: 3.6301e-02 
 This SCF iteration took 210.238 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  4 
 ========================= 
 Fermi energy = -0.189942
 AAR converged to a relative residual of 9.9703e-09 in 265 iterations.
 Poisson problem took 27.373521s
 Escc = -0.19246756
 Etot = -89.08230177
 Eatom = -29.69410059
 Error in SCF iteration: 3.1843e-02 
 This SCF iteration took 207.546 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  5 
 ========================= 
 Fermi energy = -0.189927
 AAR converged to a relative residual of 8.45366e-09 in 229 iterations.
 Poisson problem took 23.735036s
 Escc = 0.78762360
 Etot = -89.08260182
 Eatom = -29.69420061
 Error in SCF iteration: 2.9511e-02 
 This SCF iteration took 203.461 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  6 
 ========================= 
 Fermi energy = -0.184061
 AAR converged to a relative residual of 8.7359e-09 in 211 iterations.
 Poisson problem took 21.849570s
 Escc = -0.18732379
 Etot = -89.08385241
 Eatom = -29.69461747
 Error in SCF iteration: 8.2319e-03 
 This SCF iteration took 201.404 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  7 
 ========================= 
 Fermi energy = -0.183644
 AAR converged to a relative residual of 9.59773e-09 in 211 iterations.
 Poisson problem took 21.861577s
 Escc = -0.09838980
 Etot = -89.08395328
 Eatom = -29.69465109
 Error in SCF iteration: 2.3719e-03 
 This SCF iteration took 199.154 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  8 
 ========================= 
 Fermi energy = -0.184212
 AAR converged to a relative residual of 9.65304e-09 in 205 iterations.
 Poisson problem took 21.158803s
 Escc = 0.00250672
 Etot = -89.08396851
 Eatom = -29.69465617
 Error in SCF iteration: 2.5702e-04 
 This SCF iteration took 203.366 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  9 
 ========================= 
 Fermi energy = -0.184291
 AAR converged to a relative residual of 9.20537e-09 in 145 iterations.
 Poisson problem took 14.938267s
 Escc = 0.00011985
 Etot = -89.08396925
 Eatom = -29.69465642
 Error in SCF iteration: 1.4764e-04 
 This SCF iteration took 194.182 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 10 
 ========================= 
 Fermi energy = -0.184355
 AAR converged to a relative residual of 5.49146e-09 in 127 iterations.
 Poisson problem took 13.091408s
 Escc = -0.00223884
 Etot = -89.08396941
 Eatom = -29.69465647
 Error in SCF iteration: 9.7031e-05 
 This SCF iteration took 206.048 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 11 
 ========================= 
 Fermi energy = -0.184376
 AAR converged to a relative residual of 7.97446e-09 in 133 iterations.
 Poisson problem took 13.820066s
 Escc = 0.00086699
 Etot = -89.08396904
 Eatom = -29.69465635
 Error in SCF iteration: 7.4742e-05 
 This SCF iteration took 203.883 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 12 
 ========================= 
 Fermi energy = -0.184370
 AAR converged to a relative residual of 8.95156e-09 in 121 iterations.
 Poisson problem took 12.513357s
 Escc = -0.00000939
 Etot = -89.08396948
 Eatom = -29.69465649
 Error in SCF iteration: 2.2246e-05 
 This SCF iteration took 202.951 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 13 
 ========================= 
 Fermi energy = -0.184369
 AAR converged to a relative residual of 7.03254e-09 in 79 iterations.
 Poisson problem took 8.172199s
 Escc = 0.00014490
 Etot = -89.08396923
 Eatom = -29.69465641
 Error in SCF iteration: 4.8745e-06 
 This SCF iteration took 197.539 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 14 
 ========================= 
 Fermi energy = -0.184369
 AAR converged to a relative residual of 9.97338e-09 in 57 iterations.
 Poisson problem took 5.908362s
 Escc = 0.00001447
 Etot = -89.08396922
 Eatom = -29.69465641
 Error in SCF iteration: 1.5825e-06 
 This SCF iteration took 196.034 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 15 
 ========================= 
 Fermi energy = -0.184369
 AAR converged to a relative residual of 7.78418e-09 in 31 iterations.
 Poisson problem took 3.239083s
 Escc = 0.00002879
 Etot = -89.08396906
 Eatom = -29.69465635
 Error in SCF iteration: 1.2525e-06 
 This SCF iteration took 192.981 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 16 
 ========================= 
 Fermi energy = -0.184369
 AAR converged to a relative residual of 8.57892e-09 in 25 iterations.
 Poisson problem took 2.622829s
 Escc = 0.00002893
 Etot = -89.08396911
 Eatom = -29.69465637
 Error in SCF iteration: 1.1050e-06 
 This SCF iteration took 177.339 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 17 
 ========================= 
 Fermi energy = -0.184368
 AAR converged to a relative residual of 9.14341e-09 in 25 iterations.
 Poisson problem took 2.616513s
 Escc = 0.00002441
 Etot = -89.08396908
 Eatom = -29.69465636
 Error in SCF iteration: 1.0326e-06 
 This SCF iteration took 176.830 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 18 
 ========================= 
 Fermi energy = -0.184368
 AAR converged to a relative residual of 7.07588e-09 in 25 iterations.
 Poisson problem took 2.596516s
 Escc = 0.00000691
 Etot = -89.08396944
 Eatom = -29.69465648
 Error in SCF iteration: 7.1122e-07 
 This SCF iteration took 176.028 s.


 Finished SCF iteration in 18 steps!
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 

 **********************************************************
 *          Energy per unit cell = -89.083969443 Ha.       *
 *          Energy per atom = -29.694656481 Ha.            *
 **********************************************************

 Starting atomic force calculation ... 
 local force calculation: 4.006 s
 ***********************************************************
 *                      Atomic Force                       *
 ***********************************************************
 Drift free forces (Ha/Bohr):
  -0.023507881267204   0.038943382007704   0.107268522277239
   0.012522939560418  -0.011088888399928  -0.068723311797015
   0.010984941706786  -0.027854493607776  -0.038545210480224

 Max magnitude of forces (Ha/Bohr):   0.116514991977056


 Time for calculating forces: 9.215696 s.

["Stress in Ha/Bohr**2"


]  -0.000627679542602   0.000456494643031
   0.000456494643031   0.001420704211644


["Stress in GPa (equivalent to all periodic system)"


]  -0.615565655842405   0.447684535265739
   0.447684535265739   1.393285363696101


 Time for calculating stress: 27.616978 s.

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Final atomic positions (Cartesian) are as follows:
  1.972333 	  3.806260 	 15.000000 
  3.822669 	  2.034343 	 18.655220 
  4.066669 	  2.034343 	 12.344780 

 Run-time of the program: 4214.551254 seconds

ans = 

  struct with fields:

                  cell_typ: 2
                   lat_vec: [3x3 double]
                  metric_T: [3x3 double]
                    grad_T: [3x3 double]
                    lapc_T: [3x3 double]
                      Jacb: 0.866168945680640
                        L1: 6.100000000000000
                        L2: 6.100000000000000
                        L3: 30
                        Nx: 30
                        Ny: 30
                        Nz: 151
                         N: 135900
                      ecut: -1
              mesh_spacing: 0.200000000000000
                   kptgrid: [5x3 double]
                  kptshift: [0 0 0]
                      nkpt: [3 3 1]
                     tnkpt: 5
                      wkpt: [5x1 double]
                        BC: 3
                       BCx: 0
                       BCy: 0
                       BCz: 1
                      isBS: 0
                   lattice: 'undefined'
                   SCF_tol: 1.000000000000000e-06
                       Nev: 21
               poisson_tol: 1.000000000000000e-08
          pseudocharge_tol: 1.000000000000000e-08
                       Cst: 27.211386245987999
              Ha_Bohr3_GPa: 2.942101569650548e+04
                        kB: 3.166811563455600e-06
               elec_T_type: 0
                      Temp: 3.157730000000000e+02
                       bet: 1.000006412214051e+03
                       npl: 35
                       FDn: 6
                    rc_ref: 0.500000000000000
              max_relax_it: 100
                dbg_switch: 0
                        xc: 2
                 Nelectron: 26
                     n_typ: 2
                TimeRevSym: 1
                  spin_typ: 0
                     nspin: 1
              CheFSI_Optmz: 0
          chefsibound_flag: 0
               FixRandSeed: 0
                rhoTrigger: 4
                 NetCharge: 0
                 MAXIT_SCF: 100
                 MINIT_SCF: 3
             MAXIT_POISSON: 1000
            accuracy_level: -1
     target_force_accuracy: -1
    target_energy_accuracy: -1
                 TOL_RELAX: 5.000000000000000e-04
               TOL_LANCZOS: 0.010000000000000
               precond_tol: 4.088636571742514e-05
               precond_kTF: 1
            precond_thresh: 0.250000000000000
            precond_fitpow: 2
          precond_resta_q0: 1.360000000000000
          precond_resta_Rs: 2.760000000000000
            MixingVariable: 1
             MixingPrecond: 0
                  Pf_guess: []
             MixingHistory: 7
           MixingParameter: 0.300000000000000
     MixingParameterSimple: 0.300000000000000
            PulayFrequency: 1
          PulayRestartFreq: 0
                    TWtime: 1.000000000000000e+09
                 RelaxFlag: 0
                 RelaxMeth: 'LBFGS'
            max_dilatation: 1.200000000000000
            TOL_RELAX_CELL: 5.000000000000000e-06
                    MDFlag: 0
               RestartFlag: 0
                    MDMeth: 'NVE'
                     MD_dt: 1
                  MD_Nstep: 0
                     ion_T: -1
                thermos_TF: -1
              ion_elec_eqT: 1
              ion_vel_dstr: 2
                NLCG_sigma: 0.500000000000000
                     qmass: 1
                 L_history: 20
               L_finit_stp: 0.005000000000000
                  L_maxmov: 0.200000000000000
               L_autoscale: 1
                 L_lineopt: 1
                   L_icurv: 1
                   FIRE_dt: 1
                 FIRE_mass: 1
               FIRE_maxmov: 0.200000000000000
               Calc_stress: 1
                 Calc_pres: 0
            PrintForceFlag: 1
          PrintAtomPosFlag: 0
          PrintAtomVelFlag: 1
         PrintElecDensFlag: 0
            PrintEigenFlag: 0
                PrintMDout: 1
             PrintRelaxout: 1
              Printrestart: 1
           Printrestart_fq: 1
                       xin: 0
                       yin: 0
                       zin: 0
                     twist: 0
                  filename: 'MoS2'
            inputfile_path: '.'
                        XC: 'GGA_PBE'
                       Atm: [1x2 struct]
                     n_atm: 3
                     Atoms: [3x3 double]
           mvAtmConstraint: [3x3 double]
                    IsFrac: [2x1 double]
                    IsSpin: [2x1 double]
            is_default_psd: 0
                  temp_tol: 1.000000000000000e-12
                 xc_rhotol: 1.000000000000000e-14
                Atoms_init: [3x3 double]
                  lat_uvec: [3x3 double]
                        dx: 0.203333333333333
                        dy: 0.203333333333333
                        dz: 0.200000000000000
                        dV: 0.007162254770839
                        w2: [1x7 double]
                        w1: [1x7 double]
                         W: [135900x1 double]
                    occfac: 2
                Relax_iter: 1
              elecgs_Count: 1
               StressCount: 1
                    amu2au: 1.822888485332371e+03
                    fs2atu: 41.341373336493000
                 isOutl_11: [390x1 logical]
                 isOutr_11: [390x1 logical]
                      I_11: [390x1 double]
                     II_11: [390x1 double]
                      V_11: [390x1 double]
                 isOutl_22: [390x1 logical]
                 isOutr_22: [390x1 logical]
                      I_22: [390x1 double]
                     II_22: [390x1 double]
                      V_22: [390x1 double]
                      I_33: [1921x1 double]
                     II_33: [1921x1 double]
                      V_33: [1921x1 double]
                  isOutl_1: [360x1 logical]
                  isOutr_1: [360x1 logical]
                       I_1: [360x1 double]
                      II_1: [360x1 double]
                       V_1: [360x1 double]
                  isOutl_2: [360x1 logical]
                  isOutr_2: [360x1 logical]
                       I_2: [360x1 double]
                      II_2: [360x1 double]
                       V_2: [360x1 double]
                       I_3: [1770x1 double]
                      II_3: [1770x1 double]
                       V_3: [1770x1 double]
                 G_JOutl_1: [1630800x1 logical]
                 G_JOutr_1: [1630800x1 logical]
                     G_J_1: [1630800x1 double]
                     G_I_1: [1630800x1 double]
                     G_V_1: [1630800x1 double]
                 G_JOutl_2: [1630800x1 logical]
                 G_JOutr_2: [1630800x1 logical]
                     G_J_2: [1630800x1 double]
                     G_I_2: [1630800x1 double]
                     G_V_2: [1630800x1 double]
                     G_J_3: [1593000x1 double]
                     G_I_3: [1593000x1 double]
                     G_V_3: [1593000x1 double]
                   Lap_std: [135900x135900 double]
                    grad_1: [135900x135900 double]
                    grad_2: [135900x135900 double]
                    grad_3: [135900x135900 double]
                LapPreconL: [135900x135900 double]
                LapPreconU: [135900x135900 double]
              memory_usage: 1.311163208000000e+09
               staticfname: 'MoS2.static'
                 suffixNum: 0
                  outfname: 'MoS2.out'
                  parallel: 0
                         b: [135900x1 double]
                     b_ref: [135900x1 double]
                    rho_at: [135900x1 double]
                     Eself: 2.119678144426657e+02
                 Eself_ref: 4.787383725488634e+02
                       V_c: [135900x1 double]
                 PosCharge: 26.000000007599311
                 NegCharge: -26.000000007599311
                    E_corr: -2.507384708678728e-04
                       rho: [135900x1 double]
                      Atom: [1x3 struct]
                       phi: [135900x1 double]
                      e_xc: [135900x1 double]
                 dvxcdgrho: [135900x1 double]
                       Vxc: [135900x1 double]
                      Veff: [135900x1 double]
                         X: [135900x7 double]
                         F: [135900x7 double]
          mixing_hist_fkm1: [135900x1 double]
          mixing_hist_xkm1: [135900x1 double]
                       psi: [135900x21x5 double]
    upper_bound_guess_vecs: [135900x5 double]
                    EigVal: [21x5 double]
                  lambda_f: -0.184368061410350
                       occ: [21x5 double]
                    Etotal: -89.083969442691625
                     Eband: -21.030612066295951
                       Exc: -12.270350951054334
                    Exc_dc: -15.980813361072929
                  Eelec_dc: 1.402042384829668e+02
                      Eent: -2.059733055390024e-10
                      Escc: 6.911961493256599e-06
                   S_Debug: [1x1 struct]
                     force: [3x3 double]
                 abs_force: [3x1 double]
                    Stress: [2x2 double]
             stress_dimred: [2x3 double]

