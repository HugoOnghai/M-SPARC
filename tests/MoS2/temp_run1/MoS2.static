Atomic forces (Ha/Bohr):
     -0.0235078813       0.0389433820       0.1072685223
      0.0125229396      -0.0110888884      -0.0687233118
      0.0109849417      -0.0278544936      -0.0385452105
Stress (Ha/Bohr**2)                :
 -6.2767954260E-04   4.5649464303E-04 
  4.5649464303E-04   1.4207042116E-03 
Stress equiv. all periodic (GPa)   :
 -6.1556565584E-01   4.4768453527E-01 
  4.4768453527E-01   1.3932853637E+00 
