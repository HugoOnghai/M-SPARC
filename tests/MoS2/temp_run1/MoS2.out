***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 11-May-2021  Start time: 21:08:48                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 6.100000 6.100000 30.000000 
LATVEC:
1.000000000000000 0.000000000000000 0.000000000000000 
-0.500000000000000 0.866600000000000 0.000000000000000 
0.000000000000000 0.000000000000000 1.000000000000000 
FD_GRID: 30 30 150
FD_ORDER: 12
BC: P P D
KPOINT_GRID: 3 3 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.000999994
CHEB_DEGREE: 35
NSTATES: 21
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 0
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: MoS2.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.203333 (Bohr)
Mesh spacing in y-direction        :  0.203333 (Bohr)
Mesh spacing in z direction        :  0.200000 (Bohr)
Number of symmetry adapted k-points:  5
Output printed to                  :  MoS2.out
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  26
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  Mo.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.64 7.60 7.60
Number of atoms of type 1          :  1
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  S.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  7.64 7.60 7.60
Number of atoms of type 2          :  2
Estimated total memory usage       :  1.22 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9669495854E+01        1.293E-01        774.055
2            -2.9660176468E+01        3.620E-01        207.950
3            -2.9693573469E+01        3.630E-02        210.236
4            -2.9694100590E+01        3.184E-02        207.544
5            -2.9694200606E+01        2.951E-02        203.457
6            -2.9694617471E+01        8.232E-03        201.402
7            -2.9694651093E+01        2.372E-03        199.152
8            -2.9694656171E+01        2.570E-04        203.363
9            -2.9694656415E+01        1.476E-04        194.179
10           -2.9694656470E+01        9.703E-05        206.046
11           -2.9694656346E+01        7.474E-05        203.881
12           -2.9694656495E+01        2.225E-05        202.949
13           -2.9694656411E+01        4.874E-06        197.536
14           -2.9694656407E+01        1.582E-06        196.031
15           -2.9694656353E+01        1.253E-06        192.978
16           -2.9694656370E+01        1.105E-06        177.337
17           -2.9694656360E+01        1.033E-06        176.829
18           -2.9694656481E+01        7.112E-07        176.026
Total number of SCF: 18    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.9694656481E+01 (Ha/atom)
Total free energy                  : -8.9083969443E+01 (Ha)
Band structure energy              : -2.1030612066E+01 (Ha)
Exchange correlation energy        : -1.2270350951E+01 (Ha)
Self and correction energy         : -2.1196806518E+02 (Ha)
Entropy*kb*T                       : -2.0597330554E-10 (Ha)
Fermi level                        : -1.8436806141E-01 (Ha)
Average force                      :  7.8684393361E-02 (Ha/Bohr)
Maximum force                      :  1.1651499198E-01 (Ha/Bohr)
Time for force calculation         :  9.216 (sec)
Max stress                         :  1.4207042116E-03 (Ha/Bohr**2)
Max stress equiv. all periodic     :  1.3932853637E+00 (GPa)
Time for stress calculation        :  27.620 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  4214.551 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
